execution/transaction-orderer/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright © Aptos Foundation
+
```

Comment:
You could add a small description here for different things, i.e. what's batch orderer vs block-orderer vs partitioner. Usually when someone looks at the code they start from the lib.rs and see if they can find something useful there about the structure of the rest of the code.